src/commands/registries/logInToDockerCli.ts

```diff
@@ -42,8 +42,6 @@ export async function logInToDockerCli(context: IActionContext, node?: RegistryT
                     reject(new Error(`In order to log in to the Docker CLI using tokens, you currently need to go to \nOpen your Docker config file and remove "credsStore": "wincred" from the config.json file, then try again. \nDoing this will disable wincred and cause Docker to store credentials directly in the .docker/config.json file. All registries that are currently logged in will be effectly logged out.`));
                 } else if (err) {
                     reject(err);
-                } else if (stderr) {
```

Comment:
Maybe include a comment about Docker writing ignorable warnings to stderr in this case?  Is there any other reason why Docker might write to stderr (that we know about) that we would *not* want to ignore?

Reply:
None that I know of. For `docker build`, BuildKit pukes everything (not just errors) out to stderr, when I looked at telemetry of `docker build` / `docker run` there were no examples of a real failure but exit code == 0. If `docker login` is anything like `build` or `run`, we can expect real errors would always result in nonzero exit code.